tools/pythonpkg/tests/fast/arrow/test_polars.py

```diff
@@ -0,0 +1,28 @@
+import duckdb
+import pytest
+
+class TestPolars(object):
+    def test_polars(self,duckdb_cursor):
+        pd = pytest.importorskip("pandas")
```

Comment:
Not really needed to go back and forth via pandas here.

```python
df = pl.DataFrame(
    {
        "A": [1, 2, 3, 4, 5],
        "fruits": ["banana", "banana", "apple", "apple", "banana"],
        "B": [5, 4, 3, 2, 1],
        "cars": ["beetle", "audi", "beetle", "beetle", "beetle"],
    }
)
# scan plus return a polars dataframe
polars_result = duckdb.sql('SELECT * FROM df').pl()
pl.testing.assert_frame_equal(df, polars_result)

# now do the same for a lazy dataframe
lazy_df = df.lazy()
lazy_result = duckdb.sql('SELECT * FROM lazy_df').pl()
pl.testing.assert_frame_equal(df, lazy_result)

con = duckdb.connect()
con_result = con.execute('SELECT * FROM df').pl()
pl.testing.assert_frame_equal(df, con_result)
```

Reply:
Hm - that does not seem to work for me.

```py
__________________________________________________________________________________ TestPolars.test_polars ___________________________________________________________________________________

self = <test_polars.TestPolars object at 0x104b60b80>, duckdb_cursor = <duckdb.DuckDBPyConnection object at 0x104bd2df0>

    def test_polars(self,duckdb_cursor):
        pl = pytest.importorskip("polars")
        df = pl.DataFrame(
            {
                "A": [1, 2, 3, 4, 5],
                "fruits": ["banana", "banana", "apple", "apple", "banana"],
                "B": [5, 4, 3, 2, 1],
                "cars": ["beetle", "audi", "beetle", "beetle", "beetle"],
            }
        )
        # scan plus return a polars dataframe
        polars_result = duckdb.sql('SELECT * FROM df').pl()
>       pl.testing.assert_frame_equal(df, polars_result)
E       AttributeError: module 'polars' has no attribute 'testing'
```

Perhaps I need to install additional polars modules for that?

Reply:
My bad, we need to import it explicitly (that keeps import times in check):

```python
import polars as pl
import polars.testing

```

Reply:
That makes sense - thanks. I have updated the test to use Polars' native functions.